src/SlackEventHandler.ts

```diff
@@ -20,14 +20,15 @@ import { Main } from "./Main";
 import { Logging } from "matrix-appservice-bridge";
 const log = Logging.get("SlackEventHandler");
 
-interface ISlackEventChannelRenamed extends ISlackEvent {
+interface ISlackEventChannelRename extends ISlackEvent {
     // https://api.slack.com/events/channel_rename
```

Comment:
We tend to put these in `/**` comments above the interface so that editors pick up the text. They don't tend to pick up the text below the interface declaration.

Reply:
I was already thinking that this would be cool as a documentation comment.
There were two examples like this in the file, that's why I continued the style I had found.

Will change this now.